frameworks/Java/play2-java/play2-java-ebean-hikaricp-netty.dockerfile

```diff
@@ -1,5 +1,8 @@
-FROM hseeberger/scala-sbt:8u151-2.12.5-1.1.2
+FROM hseeberger/scala-sbt:8u171_2.12.6_1.1.5
 WORKDIR /play2
 COPY play2-java-ebean-hikaricp .
+
+RUN sed -i 's/.enablePlugins(PlayJava, PlayEbean, PlayNettyServer)/.enablePlugins(PlayJava, PlayEbean, PlayNettyServer).disablePlugins(PlayAkkaHttpServer)/g' build.sbt
```

Comment:
Disabling server that isn't actually going to run, this is the netty dockerfile, netty is going to act as the http server here, not akka.